docpages/02_disdppgloss.md

```diff
@@ -1,51 +1,51 @@
+* **Callback**: While not strictly related to Discord, it is used a LOT in D++. A callback is when a function is passed to another function, sort of like how you might give someone a telephone number (you give them the means to do some sort of interaction rather than asking them how to interact), which is used to handle responses to **events**.
 
-9. **Cluster**: A singular bot application, which is composed of one or more **shards**, a **cluster** is the center of bot development.
+* **Cluster**: A singular bot application, which is composed of one or more **shards**, a **cluster** is the centre of bot development.
 
-10. **\(Slash\) command**: The primary way a user interacts with a bot. It is a command sent to the bot with **parameters** (which may be optional) and is initiated by staring a message with `/`.
+* **Component**: A component is anything that can appear in a bot's message besides text, such as **buttons** and **select menus**.
 
-11. **Component**: A component is anything that can appear in a bot's message besides text, such as **buttons** and **select menus**.
+* **Drop down/Select menu**: A **component** of a message that upon being clicked drops down and allows the user to select an option.
 
-12. **Drop down/Select menu**: A **component** of a message that upon being clicked drops down and allows the user to select an option.
+* **Embeds**: A widget attached to a message which can contain multiple fields of texts, an image, and much more information.
 
-13. **Embeds**: A widget attached to a message which can contain multiple fields of texts, an image, and much more information. 
+* **Ephemeral**: A message only visible to the user being replied to.
 
-14. **Ephemeral**: A message only visible to the user being replied to.
+* **Event**: Something that a Discord bot can respond to, such as a message being sent, a **button** being clicked, or an option being selected, among others.
 
-15. **Event**: Something that a Discord bot can respond to, such as a message being sent, a **button** being clicked, or an option being selected, among others.
+* **Guild**: What the Discord API (and most libraries for it) call a server.
 
-16. **Guild**: What the Discord API (and most libraries for it) call a server.
+* **Intents**: The right for a bot to receive certain data from the Discord API.
 
-17. **Intents**: The right for a bot to receive certain data from the Discord API.
+* **Interaction**: An object that contains information about whenever a user interacts with an application, such as sending a message or clicking a button. It is the main part of an **event** that will be accessed in an application.
```

Comment:
Can we take this opportunity to fix this and clarify that a message is NOT an interaction, and a slashcommand is not a message like many beginners expect it to be

Reply:
Didn't really want to make any major contents changes in this PR, but yeah. As with the previous comment on that file above, I'm happy to put that into this changeset is wanted, or we can do that right after as a separate thing,